src/pages/wallets/find-wallet.tsx

```diff
@@ -0,0 +1,291 @@
+  hardware_support: false,
+  walletconnect: false,
+  rpc_importing: false,
+  nft_support: false,
+  connect_to_dapps: false,
+  staking: false,
+  swaps: false,
+  layer_2: false,
+  gas_fee_customization: false,
+  ens_support: false,
+  erc_20_support: false,
+  buy_crypto: false,
+  withdraw_crypto: false,
+  multisig: false,
+  social_recovery: false,
+  eip_1559_support: false,
+}
+
+export type FiltersType = typeof filterDefault
+
+export const getStaticProps = (async (context) => {
+  const { locale } = context
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPage("/wallets/find-wallet")
+  const lastDeployDate = getLastDeployDate()
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      lastDeployDate,
+    },
+  }
+}) satisfies GetStaticProps<SSRConfig>
+
+const FindWalletPage = () => {
+  const randomizedWalletData = shuffle(walletData)
+  const { pathname } = useRouter()
+  const theme = useTheme()
+  const { t } = useTranslation("page-wallets-find-wallet")
+  const resetWalletFilter = useRef(() => {})
+  const { isOpen: showMobileSidebar, onOpen, onClose } = useDisclosure()
+  const [filters, setFilters] = useState(filterDefault)
+  const [selectedPersona, setSelectedPersona] = useState(NaN)
+
+  const updateFilterOption = (key) => {
+    const updatedFilters = { ...filters }
+    updatedFilters[key] = !updatedFilters[key]
+    setFilters(updatedFilters)
+    setSelectedPersona(NaN)
+  }
+
+  const updateFilterOptions = (keys, value) => {
+    const updatedFilters = { ...filters }
+    for (let key of keys) {
+      updatedFilters[key] = value
+    }
+    setFilters(updatedFilters)
+    setSelectedPersona(NaN)
+  }
+
+  const resetFilters = () => {
+    setSelectedPersona(NaN)
+    setFilters(filterDefault)
+  }
+  
+  return (
+    <Flex direction="column" position="relative" w="full">
+      <PageMetadata
+        title={t("page-find-wallet-meta-title")}
+        description={t("page-find-wallet-meta-description")}
+      />
+
+      <BannerNotification shouldShow={true}>
+        {t("page-find-wallet-footnote-1")}
+      </BannerNotification>
+
+      <Flex
+        direction={{ base: "column-reverse", sm: "row" }}
+        position="relative"
+        w="full"
+        p={12}
+        bg="layer2Gradient"
+        mb="44px"
+      >
+        <Box w={{ base: "full", sm: "50%" }} mt={{ base: 8, sm: 0 }}>
+          <Breadcrumbs slug={pathname} />
+          <OldHeading
+            as="h1"
+            fontSize={{ base: "2.5rem", md: "5xl" }}
+            lineHeight={1.4}
+          >
+            {t("page-find-wallet-title")}
+          </OldHeading>
+          <Subtitle>
+            {t("page-find-wallet-description")}
+          </Subtitle>
+          <Subtitle>
+            {t("page-find-wallet-desc-2")}
+          </Subtitle>
+        </Box>
+        <Image
+          w={{ base: "full", sm: "50%" }}
```

Comment:
@pettinarip Can you confirm is this is appropriate here? Given this is a responsive image size, should we be using `sizes` here, paired with the `useBreakpointValue` hook?

Reply:
Not sure what's causing it, but the image here seems slightly larger than production, causing the hero section to be slightly taller than normal

https://github.com/ethereum/ethereum-org-fork/assets/54227730/80c37b5b-9717-4105-b4d3-b1c3c87846a8

Reply:
To match what is in prod, we should do 2 things:
1. Create the same node structure that we have in prod (the Gatsby image component renders the image with a wrapper, while Next image renders only the img tag).
2. Crop the image with the same size as we do in the Gatsby query (600px in this case).

What I've been doing recently would be something like this:
```tsx
// wrap the img and set the layout styles in the wrapper
<Center w={{ base: "full", sm: "50%" }}>
  <Image
    src={FindWalletHeroImage}
    // define a image width as we are doing in the Gatsby queries
    width={600}
    alt=""
    priority
    style={{
      objectFit: "contain",
    }}
  />
</Center>
```

Reply:
Pushed a commit with those changes.